Firestore/Swift/Source/Codable/Errors.swift

```diff
@@ -0,0 +1,16 @@
+//
```

Comment:
Copyright notice.

I realize this isn't super important to you, but this is the only way that Google allows me to continue to work in the open with you, so it's very important to me :-).